EBPF_LICENSE.txt

```diff
@@ -0,0 +1,342 @@
+The eBPF code loaded by Prodfiler into the kernel is covered by the GPLv2 below:
```

Comment:
Can we leave this comment away and stay with the pure GPLv2 license text ?
The HA -copyright says it and then just links to this copy of the GPLv2 text.

Given that someone asks for the ebpf source code, we have to hand it out together with the license text. Else it stays unclear under what license the files are. Maybe (additionally) mention the license in each file's header comment !?

Reply:
The ebpf files *should* also have a Copyright note.
And does it make sense to link from the HA to the GPLv2 text on the gnu.org website instead linking to our own copy ?

Reply:
I'll remove the first line from EBPF_LICENSE.txt. Updating ebpf source should be a separate PR (ideally, also containing code to generate the deliverable source code archive).

Reply:
> And does it make sense to link from the HA to the GPLv2 text on the gnu.org website instead linking to our own copy ?

I prefer linking to our own copy for two reasons:

+ We get to dictate exactly what the license is and where it's placed.
+ By virtue of naming (EBPF_LICENSE.txt) we make it even more clear that the license only applies to the ebpf parts and not to prodfiler as a whole.

Reply:
> (ideally, also containing code to generate the deliverable source code archive)

Are you going to work on this or should we open a tracking PR ?

Reply:
I opened https://optimyze2.myjetbrains.com/youtrack/issue/PF-1836